{"name": "test_one", "status": "skipped", "statusDetails": {"message": "XFAIL 0不能做被除数\n\nZeroDivisionError: division by zero", "trace": "@pytest.mark.xfail(reason=\"0不能做被除数\")\n    def test_one():\n>       1 / 0\nE       ZeroDivisionError: division by zero\n\ntestcases/test_xfail.py:7: ZeroDivisionError"}, "start": 1717683261443, "stop": 1717683261443, "uuid": "07176eb9-f9f0-4eac-900c-90d61b968660", "historyId": "cc15ce69b6cffd555b1d3f2e124aa7de", "testCaseId": "cc15ce69b6cffd555b1d3f2e124aa7de", "fullName": "testcases.test_xfail#test_one", "labels": [{"name": "tag", "value": "@pytest.mark.xfail(reason='0不能做被除数')"}, {"name": "parentSuite", "value": "testcases"}, {"name": "suite", "value": "test_xfail"}, {"name": "host", "value": "hefangdeMacBook-Pro.local"}, {"name": "thread", "value": "61233-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testcases.test_xfail"}]}